test/testdata/lsp/completion/splat_call.rb

```diff
@@ -16,13 +16,13 @@ def sumup_splat(*as)
 sig { params(avec: T::Array[A], blk: T.proc.params(a0: A).returns(T.untyped)).returns(T.untyped) }
 def call_with_block(avec, &blk)
   avec.su(&blk) # error: does not exist
-#        ^ completion: sum, sum, sum, sum
+#        ^ completion: sum
```

Comment:
cc @froydnj, no more `sum, sum, sum, sum`!

Reply:
That about `sum`s it up!